src/test/java/com/commercetools/emailprocessor/email/EmailProcessorTest.java

```diff
@@ -103,32 +107,32 @@ public void processEmail_pendingEmailAvailable_shouldProcessEmails() throws Exce
     @Test
     public void processEmail_processAllFlagIsSet_shouldProcessAllEmails() throws Exception {
         customObjects.add(createCustomObject("1", STATUS_PENDING, RESPONSE_CODE_SUCCESS));
-        customObjects.add(createCustomObject("2", STATUS_PENDING, Statistics.RESPONSE_ERROR_TEMP));
-        customObjects.add(createCustomObject("3", EMAIL_STATUS_ERROR, Statistics.RESPONSE_ERROR_TEMP));
+        customObjects.add(createCustomObject("2", STATUS_PENDING, RESPONSE_ERROR_TEMP));
+        customObjects.add(createCustomObject("3", EMAIL_STATUS_ERROR, RESPONSE_ERROR_TEMP));
         customObjects.add(createCustomObject("4", STATUS_PENDING, Statistics.RESPONSE_ERROR_PERMANENT));
-        tenantConfiguration.setClient(mockSphereClient(customObjects));
+        tenantConfiguration.setClient(mockSphereClient(customObjects, false));
```

Comment:
but the method name contains `..processAllFlagIsSet..`, and you are passing `false`?

Reply:
5dfb7016e3f7338c560a9116fca9e69fdd1186ac